src/style.css

```diff
@@ -1,121 +1,121 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
-
-body {
-  background-color: rgba(0, 0, 0, 0.5);
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  height: 100vh;
-  font-family: 'Roboto Mono', monospace;
-}
-
-.main-section {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  /* stylelint-disable-next-line csstree/validator */
-  height: fit-content;
-  min-width: 50%;
-  margin-inline: auto;
-  background-color: #fff;
-  box-shadow: 1px -3px 5px 0 rgba(0, 0, 0, 0.52);
-  border-radius: 3px;
-}
-
-.container {
-  display: flex;
-  flex-direction: column;
-  min-width: 100%;
-  height: 100%;
-}
-
-.heading,
-.input-items {
-  display: flex;
-  justify-content: space-between;
-  padding: 1rem;
-  border-bottom: 1px solid rgba(128, 128, 128, 0.199);
-}
-
-.list-item {
-  list-style: none;
-  display: flex;
-  justify-content: space-between;
-  padding: 1rem;
-  border-bottom: 1px solid rgba(128, 128, 128, 0.199);
-}
-
-.todolist {
-  padding-left: 0%;
-}
-
-.task {
-  margin-left: 5px;
-}
-
-.clear {
-  text-align: center;
-  padding: 1rem;
-  background: rgba(128, 128, 128, 0.112);
-}
-
-.clear-list {
-  border: none;
-  cursor: pointer;
-  background-color: transparent;
-  color: rgba(128, 128, 128, 0.92);
-  font-family: 'Roboto Mono', monospace;
-  transition: 0.4s;
-}
-
-.clear-list:hover {
-  color: rgba(0, 0, 0, 0.5);
-}
-
-.input-text {
-  border: none;
-}
-
-.input-text:focus {
-  outline: none;
-}
-
-i {
-  color: rgba(128, 128, 128, 0.92);
-  cursor: pointer;
-}
-
-#ellipsis {
-  cursor: move;
-  display: inline;
-}
-
-.backyellow {
-  background-color: rgba(255, 217, 0, 0.169);
-}
-
-.inline {
-  display: inline;
-}
-
-.fa-trash-can {
-  display: none;
-}
-
-.fa-trash-can:hover {
-  color: red;
-}
-
-input[type="checkbox"] {
-  height: 20px;
-  width: 20px;
-  color: green;
-}
-
-.checkbox:checked::before {
-  background-color: green;
-}
-
-.fa-ellipsis-vertical {
-  cursor: move;
-}
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
+
+body {
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: 'Roboto Mono', monospace;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  /* stylelint-disable-next-line csstree/validator */

```

Comment:
Implementing linters is enforced to make our code clean and follow industry standards. I think instead of disabling linters, finding ways to solve linter errors would be a much better solution.